### ISSUE TITLE
Playing an in game video,will crash spectators game during replay.

### ISSUE THREAD

**Issue by KOYK** [OpenRA/OpenRA#13630]
Prof of bug
http://resource.openra.net/maps/21800/

Note*
add full number of bots during test of the bug on this map,otherwise lua will crash.

**Comment by abcdefg30** [comment]
Even if it doesn't crash, the game is still paused for the players without the video for no obvious reason. We should try to fix that as well.

**Comment by pchote** [comment]
> for no obvious reason

The game is paused while the video is playing, and everyone needs to stay in sync...?

**Comment by abcdefg30** [comment]
Yes I know that. "for no obvious reason" as in "the player ingame doesn't know what is going on (see my edit).

**Comment by abcdefg30** [comment]
We could add some simple text saying that a video is playing for player xy as "workaround".

**Comment by KOYK** [comment]
Aside from adding a massage to the user to clarify that some one is watching a video,
the other player must be stopped from unpausing the game, it will need some short of "lock" the pause state until both users are ready to play, 
other wise one player will start playing the game and the other will continue to watch the video(and probably get confused about the rifle sounds in the background :p )

Maybe wait for all human players to automatically unpause the game at the end of the video.

_Or add a mini-game for the ones that don't have the videos_(**just kidding**)

**Comment by KOYK** [comment]
It wont be fixed\added, closing it.

**Comment by pchote** [comment]
If this is still a problem then we are going to need to fix it for Allies 10a (#19336).

**Comment by penev92** [comment]
~~I copied the video-playing part from Allies 10a (#19336) over to Fort Lonestar and started a multiplayer game with 1 client as player and one client as spectator. Both clients played the video correctly and the game continued normally. All fine when I watched both generated replays later (video plays again). Am I missing something?~~
Ok so there is no crashing, but I see the problem with unpausing in multiplayer (by spectators or players). I don't think this affects missions though, since there is only one player there.

**Comment by pchote** [comment]
To confirm, we don't need this for the current milestone then?

**Comment by abcdefg30** [comment]
Removed it from the milestone.
